common/djangoapps/util/keyword_substitution.py

```diff
@@ -53,30 +46,36 @@ def substitute_keywords(string, user=None, course=None):
     the loop below, and reduces the need for piling up if elif else statements
     when the keyword pool grows.
     """
```

Comment:
You can't see it in this diff, but the last four lines of the docstring can be removed now since they don't apply anymore